lib/pages/home/widgets/features.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:school_erp/theme/colors.dart';
+import '../../../theme/text_styles.dart';
+import '../../../features/auth/auth_repository/schemas/user.dart';
+import '../../EnterExitRoute.dart';
+import '../../profile/profile_page.dart';
```

Comment:
i think prefer to use abs import for explicitness / clearness. Especially for the case of going 3 levels up